docs/node-operators/get-started/setup-datanode.md

```diff
@@ -76,6 +76,86 @@ Where:
 - `/host_path/to/snapshotsCopyTo` is the path on your host machine where you want to store the snapshots that are generated by data node. Snapshots allow you to restore the database to a previous state.
 - `/host_path/to/snapshotsCopyFrom` is the path on your host machine where you want to store the snapshots that are retrieved from IPFS and can be used to rebuild a data node database from another data node.
 
+### Postgresql configuration tuning
+
+The default PostgreSQL configuration is not optimized for memory usage. We have to modify a few PostgreSQL parameters in the `postgresq.conf` file. The default parameter file path for Linux system and PostgreSQL 14 is: `/etc/postgresql/14/main/postgresql.conf`.
+
+Suggested parameters for the `PostgreSQL` are:
+
+#### `max_connections`
+
+New value:
+
+```conf
+max_connections = 50
+```
+
+We should limit the maximum number of connections as it directly impacts the memory usage by PostgreSQL.
+
+#### `huge_pages`
+
+New value:
+
+```conf
+huge_pages = off
+```
+
+The default value of the `huge_pages` config is `try`. Setting it to `off` usually reduce the RAM usage, but it increases the CPU usage on the machine.
+
+#### `work_mem`
+
+New value:
+
+```conf
+work_mem = 5MB
+```
+
+#### `temp_buffers`
+
+New value:
+
+```conf
+temp_buffers = 5MB
+```
+
+#### `shared_buffers`
+
+New value:
+
+```conf
+shared_buffers = 2GB
+```
+
+New value depends on your server memory. It should be set to `25%` of the system physical memory. The `2GB` value should be for the server with `8GB` physical memory.
+
+#### `dynamic_shared_memory_type`
+
+New value:
+
+```conf
+dynamic_shared_memory_type = sysv
+```
+
+#### `shared_memory_type`
+
+New value:
+
+```conf
+shared_memory_type = sysv
+```
+
+
+#### Total memory usage
```

Comment:
I don’t think this section is talking about changing configuration, but rather helping someone figure out they can calculate memory. If there is config to change, it could be clearer. And if not, this subsection can be nested outside of the config section.